static/js/libs/throttle.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Throttle
+ * @param {Function} func Function to run.
+ * @param {Number} wait Time to wait between tries.
+ */
+
+export default function throttle(func, wait) {
+  let time = Date.now();
```

Comment:
`time` shouldn't be set to `Date.now()` by default in `throttle`. It only makes sense it `func` is called immediately (which this throttle doesn't do by itself).

```suggestion
  let time;
```

Reply:
👍

Reply:
Or we can introduce `immediate` parameter - similar to our `debounce` implementation to set up throttle and immediately call function.